### ISSUE TITLE
pip installing version 0.0.0

### ISSUE THREAD

**Issue by Shandelier** [tensorflow/decision-forests#3]
Problem:
Installing via `pip install tensorflow-decision-forests` returns warning message:
<details>
<summary>Warning message</summary>
<br>
```
WARNING: The candidate selected for download or install is a yanked version: 'tensorflow-decision-forests' candidate (version 0.0.0 at https://files.pythonhosted.org/packages/9b/c2/5b5d8796ea5cb8c19457cd9b563c71b536ffc3d14936049fa9cf49b31dcf/tensorflow_decision_forests-0.0.0-py3-none-any.whl#sha256=18ed810f415437ef8e8a4848d3cbf2cff51e3ea2b66224b75f9d9c0f689629a7 (from https://pypi.org/simple/tensorflow-decision-forests/))
Reason for being yanked: <none given>
```
</details>

After installation the library can't be used because of:
<details>
<summary>Warning message</summary>
<br>
```
Traceback (most recent call last):
  File "c:/Users/kluse/Documents/python/how-to-active-learning/main.py", line 5, in <module>  
    import model as mdl
  File "c:\Users\kluse\Documents\python\how-to-active-learning\model.py", line 12, in <module>
    import tensorflow_decision_forests as tfdf
ModuleNotFoundError: No module named 'tensorflow_decision_forests'
```
</details>

Specifying version via `==0.1.3` doesn't help, and reinstalling too.

**Comment by arvnds** [comment]
Hi -- thanks for the error report!

Based on your filepath, I'm going to assume you are trying to install on Windows, please correct me if I'm wrong.

Unfortunately we have not released a pip package on Windows yet, we're working on it! As a result PyPi is falling back to a default empty version.

This is documented here: https://github.com/tensorflow/decision-forests/blob/main/documentation/known_issues.md and we will update that page hopefully soon with a Windows compatible release.

Hope that helps!

**Comment by Shandelier** [comment]
Thanks for reply. I've edited issue title. 
Waiting for Windows release! :) In the meantime i'll try out Linux

**Comment by Vlek** [comment]
I have tried using this package on Linux, I'm getting GPU errors so I tried on both on Windows and Mac laptops. Both are getting the 0.0.0 versions.

**Comment by achoum** [comment]
The "yanked" error message was indicative that no compatible versions were available (OS, OS version or python version). I removed the yanked version (which was a placeholder). The error message should be more explicit now.

The list of available versions is listed [here](https://pypi.org/manage/project/tensorflow-decision-forests/release/0.1.3/).
TLDR: ManyLinux{2_12,2010} + Python 3.{6,7,8,9}.

This issue will be resolved once a Windows version is made available (WIP).

> Vlek

You mentioned a GPU error on Linux. Would you mind creating an issue with a copy of the logs and the version of linux+python you are using? :)

**Comment by miguel-dsv** [comment]
Hi, thanks a lot for creating this great package!. Is there an expected timeline for the Windows release? Thanks!
